src/legacy/core_plugins/data/public/filter/filter_bar/filter_bar.tsx

```diff
@@ -109,7 +109,7 @@ class FilterBarUI extends Component<Props, State> {
   }
 
   private renderAddFilter() {
-    const isPinned = this.props.uiSettings.get('filters:pinnedByDefault');
+    const isPinned = this.props.uiSettings && this.props.uiSettings.get('filters:pinnedByDefault');
```

Comment:
@lizozom this removes the error but, doesn't fix the origin of this error, it might have to do with the NP migration, do you have an idea, why uiSettings is `undefined` only in the FilterBar of Context?